### ISSUE TITLE
Expiration reminder time incorrect when pairing new pod

### ISSUE THREAD

**Issue by ryangs23** [LoopKit/Loop#1609]
When pairing a new pod, the default/suggested pod expiration reminder date and time is the same as that of the previously paired pod, even if this time is in the past. See attached screenshot where suggested reminder time is today at 17:26, even though the current time is 19:04.

Occurs with Loop v2.3.0 (57), built from a fresh clone of dev branch on January 19, 2022. Eros pods.

[See discussion on ZulipChat.](https://loop.zulipchat.com/#narrow/stream/144182-development/topic/Pod.20expiration.20reminders)

![IMG_7277](https://user-images.githubusercontent.com/50974083/155407741-4333ece4-88ea-417d-ae26-caa9eb58cadb.PNG)

**Comment by loudestnoise** [comment]
I've seen this issue as well with `dev` branch of DIY Loop on my iPhone. I've been meaning to file an issue, but hadn't gotten around to it. I've mostly just ignored the incorrect expiration reminders, but now that I know you're seeing this as well I know it's not just me. I don't think it always happens to me, but maybe now I'll pay more attention to when it happens. I just paired a new pod today and it looks like it's got a proper expiration reminder of 24 hours before the pod expires ¯\_(ツ)_/¯ .

![image](https://user-images.githubusercontent.com/194853/155410071-aee63bca-ce3b-4a7b-b530-613ce7a1e280.png)

**Comment by ryangs23** [comment]
I seem to have resolved this issue by rebuilding Loop from a fresh clone of dev branch on Feb 23 2022 at 17:15 EST. As part of this rebuild, I deleted my old Loop app. So it's not clear if this issue was resolved simply by the updated version, or if deleting all my old settings helped (or was the only thing required). Either way, I have now paired two pods and the suggested expiration reminder was when I expected: two hours before the 72-hour mark.
